release-notes/release-notes509.md

```diff
@@ -5,6 +5,8 @@ title_nav: TinyMCE 5.0.9
 keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissues
 ---
 
+> **Important:** Premium Plugins where updated on the 17th July, 2019, as described in the [Errata&#58; TinyMCE 5.0.9 Premium Plugins]({{site.baseurl}}/release-notes/release-notes509/#erratatinymce509premiumplugins).
```

Comment:
You don't need the full URL here since it's in the same page, so this can just be shortened to `[Errata&#58; TinyMCE 5.0.9 Premium Plugins](#erratatinymce509premiumplugins)`

Reply:
Also `s/where/were/`?

Reply:
Amended Commit to fix issue.